daisy/workflow/workflow.go

```diff
@@ -336,7 +336,6 @@ func (w *Workflow) populate(ctx context.Context) error {
 func (w *Workflow) NewIncludedWorkflow() *Workflow {
 	iw := New()
 	iw.Cancel = w.Cancel
-	iw.parent = w
```

Comment:
It doesn't NEED it. However, your logger uses it to build "grandparent.parent.child" log prefixes.